src/libprojectM/TimeKeeper.cpp

```diff
@@ -95,12 +95,12 @@ int TimeKeeper::PresetFrameA()
     return m_presetFrameA;
 }
 
-int TimeKeeper::PresetTimeB()
+double TimeKeeper::PresetTimeB()
```

Comment:
These had an int return value? Huh. Never noticed that, makes no sense, right?

Reply:
Yeah, I was surprised too. One of the few places the type mismatch might matter.